QUICKSTART-EKS.md

```diff
@@ -156,7 +158,11 @@ aws ssm get-parameters --region us-west-2 \
 
 ### Cluster setup
 
-*Note:* most commands will have a region argument; make sure to change it if you don't want to set up in us-west-2.
+*Note:* most commands will have a region argument; make sure to change it if you don't want to set up in us-west-2. Also be aware that when operating in GovCloud the IAM ARNs will need to be updated to the following: `arn:aws-us-gov`.   
```

Comment:
New sentences should start on a newline - markdown will render them as part of the paragraph as long as there are no blank lines between the sentences.  For example:
```
A sentence.
Another sentence
```

should render like so:
```
A sentence.  Another sentence.
```